### DISCUSSION TITLE
Slider

### DISCUSSION THREAD

**Discussion by PabloFNK** [twilio-labs/paste#976] [Ideas] [answered]
We are using a Material-UI slider and it would be nice to have one from Paste

**Comment by SiTaggart** [comment, marked as answer]
Hi @PabloFNK 

A slider is on our [component roadmap](https://paste.twilio.design/roadmap), but it just hasn't been scheduled into any release yet. This is likely due to not seeing a strong demand for it in our roadmap reviews with BUs and Product Designers.

Roadmap reviews are conducted every quarter and are used to help reprioritize that roadmap backlog based on what BUs are going to need. So if we start seeing a strong need across multiple BUs it'll likely start making its way up the release plan.

Contribution is also always an option. Your Product Design partner is encouraged to contribute to the system, as well as your engineering team. There is a fairly robust workflow in place to follow, and it is a lot of work, but we're open to supporting teams contributing to the system.

We've seen component contributions in the past take 10-12 weeks to complete.

**Reply by PabloFNK** [reply]
We don't have a rush on this so knowing that it is on the roadmap (even if it is not scheduled) is enough for us :).
